examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts

```diff
@@ -90,4 +94,20 @@ export class BpiAccountStorageAgent extends PrismaService {
       where: { id: bpiAccount.id },
     });
   }
+
+  async storeAccompanyingStateLeafValues(
+    id: string,
+    leafIndex: number,
+    merkPayload: MerkleTree,
+    witness: Witness,
+  ): Promise<void> {
+    await this.bpiAccountStateTreeLeafValue.create({
```

Comment:
bpiAccountStateTreeLeafValue class been implemented?

Reply:
Yes, if you are refering to the prisma model.